examples/MamaDuckVerbose/MamaDuckVerbose.ino

```diff
@@ -37,33 +44,54 @@ void loop() {
   
   //Handle OTA requests
   ArduinoOTA.handle();
+  
+  //Get flag if LoRa packet received
+  if(duck.getFlag()) {
+
+    //Flip lora flag for next message
+    duck.flipFlag(); 
 
-  if(duck.getFlag()) {  //Get flag if LoRa packet received
-    duck.flipFlag(); //Flip lora flag for next message
-    duck.flipInterrupt(); //Flip interrupt to false
-    int pSize = duck.handlePacket(); //Read and store lora packet
+    //Flip interrupt to false
+    duck.flipInterrupt(); 
```

Comment:
I am assuming we disable the interrupts so we can safely process the received packet.